addons/knobs/src/components/types/Options.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactSelect from 'react-select';
+import styled from '@emotion/styled';
+
+import RadiosType from './Radio';
+import CheckboxesType from './Checkboxes';
+
+const OptionsSelect = styled(ReactSelect)({
+  width: '100%',
+  maxWidth: '300px',
+  color: 'black',
```

Comment:
I have pushed this fix for the `ReactSelect` text color issue in the dark theme - [see updated deployment](https://deploy-preview-4723--storybooks-official.netlify.com/?selectedKind=Addons%7CKnobs.withKnobs&selectedStory=optionsKnob&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)